helm/solr-operator/templates/deployment.yaml

```diff
@@ -64,6 +64,10 @@ spec:
         {{- if .Values.mTLS.insecureSkipVerify }}
         - --tls-skip-verify-server={{ .Values.mTLS.insecureSkipVerify }}
         {{- end }}
+        {{- if .Values.mTLS.watchForUpdates }}
+        - --tls-watch-cert={{ .Values.mTLS.watchForUpdates }}
```

Comment:
`{{- if .Values.mTLS.watchForUpdates }}` returns false for `false` values, so the only way for the if statement to be passed is if `watchForUpdates` is true. Since the default value in the operator is `true`, there's no way to set `watchForUpdates` to *false* using the helm chart.

I would probably just remove the if statement here.

Reply:
ha ha! good catch, was just copy-and-pasting senselessly ...

Reply:
verified by doing:
```
helm template helm/solr-operator --set mTLS.watchForUpdates=false
```
resulting config is:
```
       args:
        - -zk-operator=true
        - --tls-skip-verify-server=true
        - --tls-watch-cert=false
```